assets/css/style.css

```diff
@@ -0,0 +1,136 @@
+/* font styling */
+h1,
+h2,
+h3,
+h4,
+p,
+ul,
+li {
+font-family: 'Open Sans', sans-serif;
+}
+p {
+  font-size: 1.25em;
+  color: gray;
+  line-height: 32px;
+}
+
+h1 {
+  font-size: 40px !important;
```

Comment:
Yea... sometimes it is necessary. Typically something like this is caused by the order in which you're loading your files. So if your order is:

1. Custom CSS
2. Bootstrap CSS

The Bootstrap styles will overwrite your Custom styles. Get your Custom CSS to load last and it *usually* works fine.

Again, sometimes you just have to use `!important` but it's usually seen as a bandaid fix.